src/Application/Components/ClickCanvas/ClickCanvas.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { fabric } from 'fabric-webpack';
+import numeral from 'numeral';
+
+const ClickCanvas = React.createClass({
+  propTypes: {
+    cash: React.PropTypes.number.isRequired,
+    income: React.PropTypes.number.isRequired,
+    onClick: React.PropTypes.func.isRequired,
+    image: React.PropTypes.string.isRequired,
+  },
+
+  images: {
+    penny: 'images/penny.gif',
+  },
+
+  getInitialState() {
+    return {
+      cash: 0,
+      income: 0,
+    }
+  },
+
+  _createText(x, y, color, text) {
+    return new fabric.Text(text, {
+      left: x,
+      top: y,
+      fill: color,
+      angle: 0,
+      fontSize: 20,
+      selectable: false,
+    });
+  },
+
+  setClickableImage() {
+    fabric.Image.fromURL(this.images[this.props.image], img => {
+      img.set({
+        left: this.imgX,
+        top: this.imgY,
+        selectable: false,
+      });
+      if (this.clickable) {
+        this.canvas.remove(this.clickable);
+      }
+      this.clickable = img;
+      this.clickable.on('mousedown', this.props.onClick);
+
+      this.canvas.add(this.clickable).renderAll();
+    });
+  },
+
+  setupDefaults() {
+    this.imgX = 250;
+    this.imgY = 50;
+    this.clickable = undefined;
+
+    const x = 50;
+    const y = 50;
+
+    this.moneyLabel = this._createText(x, y, '#000', 'Cash: ');
+    this.mpsLabel = this._createText(x, y + 20, '#000', 'Income: ');
```

Comment:
Fair, except that it says 'income: ' right there. I'll change it, because it makes sense, but FUCK YOU.